app/styles/hypertable.less

```diff
@@ -13,7 +13,6 @@
   display: inline-grid;
   grid-auto-flow: column;
   grid-template-columns: 390px repeat(var(--hypertable-responsive-columns-number, 0), 1fr);
-  grid-auto-flow: column;
```

Comment:
you're sure this doesn't break the horizontal scrolling in the table? 🤔

Reply:
This line is defined two line up. I just remove the duplicate